tests/integration/workflows/python_pip/test_python_pip.py

```diff
@@ -28,31 +29,40 @@ def setUp(self):
         self.builder = LambdaBuilder(language="python",
                                      dependency_manager="pip",
                                      application_framework=None)
+        self.runtime = "{language}{major}.{minor}".format(
```

Comment:
nice!! this makes the test work across multiple runtimes :)

Reply:
Yes! :)